src/com/palmergames/bukkit/towny/object/WorldCoord.java

```diff
@@ -299,12 +299,24 @@ public static boolean cellChanged(Location from, Location to) {
 			   !Objects.equals(from.getWorld(), to.getWorld());
 	}
 
-	public List<WorldCoord> getCardinallyAdjacentWorldCoords() {
-		List<WorldCoord> list = new ArrayList<>(4);
+	public List<WorldCoord> getCardinallyAdjacentWorldCoords(boolean... includeOrdinalFlag) {
+		boolean includeOrdinal = (includeOrdinalFlag.length >= 1) ? includeOrdinalFlag[0] : false;
+		List<WorldCoord> list;
+		if (includeOrdinal) {
+			list = new ArrayList<>(8);
+		} else {
+			list = new ArrayList<>(4);
+		}
 		list.add(this.add(0,-1));
 		list.add(this.add(0,1));
 		list.add(this.add(1,0));
 		list.add(this.add(-1,0));
+		if (includeOrdinal) {
+			list.add(this.add(1,1));
+			list.add(this.add(1,-1));
+			list.add(this.add(-1,-1));
+			list.add(this.add(-1,1));
+		}

```

Comment:
Unneeded diff as far as I can tell.

Reply:
Nevermind....

Reply:
The change allows this function to detect diagonal (NE,NW,SE,SW) ownership, otherwise known as "Ordinal compass directions". It doesn't change the default functionality of only Cardinal directions (N,E,S,W).